Common/Global.cs

```diff
@@ -351,6 +351,11 @@ public enum SecurityType
         /// Contract For a Difference Security Type.
         /// </summary>
         Cfd,
+        
+        /// <summary>
+        /// Index Security Type.
+        /// </summary>
+        Index,
```

Comment:
To preserve the numerical value of the enum we should put this at the end of the enum definition